src/index.html

```diff
@@ -8,16 +8,42 @@
     <link rel="stylesheet" href="styles/styles.css" />
   </head>
   <body>
-    <header class="app-header">
-      <h1 class="app-header__heading">Quiz App</h1>
+    <header class="header">
+      <div class="flex mx-4 my-1 flex-align-center">
+        <img class="header__profile" src="../../assets/images/profile-icon.png" alt="" />
+        <h1>Hello, <strong>{User}</strong></h1>
+        <span class="header__dots">...</span>
+      </div>
     </header>
 
     <section class="quiz-card mx-4 my-4 px-3 py-3">
```

Comment:
The quiz-cards should be the main content on your website and therefore wrapped in a `main` element.